### ISSUE TITLE
Add-on lists: show add-on versions changes [Preferences] to [Options]

### ISSUE THREAD

**Issue by 5t3f4n** [Aris-t2/ClassicThemeRestorer#161]
Before/After
![Before](https://i.imgur.com/XpC43hg.png)
![After](https://i.imgur.com/3k3sLJC.png)

`Classic Theme Restorer 1.4.7`
`Firefox 44.0.2`
`OS X 10.11.3 (15D21)`

**Comment by Aris-t2** [comment]
This is a known minor "issue" between Windows ("Options" labels) and Linux/MacOSX ("Preferences" labels). I had to chose one of them and because the majority for CTR users are Windows users (88%), I have chosen the "Options" label. Otherwise I would have to add two duplicate files with over 1000 lines of code each with only one difference in one line: the label reference.

I would prefer not to increase CTRs file size for such a reason.

**Comment by Aris-t2** [comment]
Just tested:
current add-on file without label fix: 672KB
new add-on file with label fix: 691KB

Not sure, if fixing one label is worth 19KB of additional size, what do you think?

**Comment by 5t3f4n** [comment]
>I had to chose one of them

Can you explain this? Wouldn't it require *less* code to not change it at all?

>Not sure, if fixing one label is worth 19KB of additional size, what do you think?

I would say absolutely. I would never consider file size being more valuable than a bugfix. Personally I wouldn't care if the file size increased by 500%. What does it matter really?

**Comment by Aris-t2** [comment]
Mozilla uses two different labels for the same button (one on Windows and one on Linux/MacOSX) instead of using one and only altering its referenced OS-specific target. CTR can of course use the same file for every OS or different files for every OS. In the case the file would only differ in one label, the rest of the code (1000-1200 lines) is the same. Normally this would not be a problem, because compressing files to e.g. rar or 7z would only recompress the part that differs, but Firefox uses xpi files that are normal zip files without features like rar/7z have.

Long story short: in my opinion 19kb of file size is "nothing", but if people will complain, I have to rethink that fix. Next beta will already contain it. ;-)

**Comment by 5t3f4n** [comment]
>Mozilla uses two different labels for the same button (one on Windows and one on Linux/MacOSX) instead of using one and only altering its referenced OS-specific target

I'm with you this far, but what I don't get is why CTR has to change either of those labels at all? It should just change `Classic Theme Restorer` into `Classic Theme Restorer 1.4.7` and nothing else.

* Alternative appearnce
* Add-on lists: hide warnings
* Add-on lists: compact view
* Add-on lists: highlight entry on hover
* 'More info' page: alternative order
* Old add-on install window

None of these other options change it from Preferences to Options.

**Comment by Aris-t2** [comment]
In order to add the version number the whole add-on entry incl. buttons/labels/order etc. gets reset to an older version. The xml file (to build the add-on entry) containing the "old information" is from Windows, so for that button it uses the Windows label, which is "Options" in the English version of Firefox. Most locales except English use the same label cross platform.

To sum up: only MacOSX and Linux users of the English Firefox versions are affected by the "Options"-"Preferences" mixup. I doubt many people even notice that.

**Comment by Aris-t2** [comment]
Fixed in CTR 1.4.8
